docs/ABI.md

```diff
@@ -0,0 +1,1176 @@
+orphan
+
```

Comment:
This looks like detritus from the automatic conversion. You might need to check how other `..` annotations got mangled too.